### DISCUSSION TITLE
nav tabs not working

### DISCUSSION THREAD

**Discussion by Voctor-R** [twbs/bootstrap#37108] [Q&A]
Hello,

I'd like some help to get the nav tab working. I think I've set up everything fine (but obviously not) and I don't know where's the problem coming from. 

What I'd like to achieve is something like this [https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior](url) to have multiple tabs to show each different tab.

Here's the HTML

`<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta http-equiv="X-UA-Compatible" content="ie=edge">
	<title>Test</title>
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.0/dist/bootstrap-table.min.css">
	<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.13.2/themes/smoothness/jquery-ui.css">
</head>

<body>
	<main>
		<intro>
			<p class="intro">Text</p>
		</intro>

		<div class="tab-content" id="myTabContent">
			<div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab">
				<table id="mytable" rules=groups frame=box hborder=0 cellspacing=1 cellpadding=4 vborder=1>
					<thead class="table_header">
						<th id="colex">Co<br />Ex</th>
						<th id="h_M8">V<br />M8</th>
						<th id="h_M1">cd<br />M1</th>
						<th id="h_M2">N°<br />M2</th>
						<th id="h_M4">Désignation<br />M4</th>
						<th id="h_M7">Uté<br />M7</th>
						<th id="h_M9">Qté <br />M9</th>
						<th id="h_M18">PR Fourniture unitaire<br />M18</th>
						<th id="h_M20">Type FO<br />M20</th>
						<th id="h_M14">K Acc<br />M14</th>
						<th id="h_M19">PR Total fourniture<br />M19</th>
						<th id="h_M23">Tps pose unitaire<br />M23</th>
						<th id="h_M26">Type MO<br />M26</th>
						<th id="h_M22">K Pose<br />M22</th>
						<th id="h_M42">Tps pose total<br />M42</th>
						<th id="h_M40">PR unitaire<br />M40</th>
						<th id="h_M41">PR Total<br />M41</th>
						<th id="h_M27">qte vendue<br />M27</th>
						<th id="h_M32">PV unitaire<br />M32</th>
						<th id="h_M44">PV Total<br />M44</th>
					</thead>
					<tbody>
						<tr class="parent" data-level="0">
							<td headers="colex" class="colex">-</td>
							<td headers="h_M8"><input class="variante" type="checkbox"></td>
							<td headers="h_M1">1</td>
							<td headers="h_M2">A</td>
							<td headers="h_M4">Titre 1, n'appartient à personne</td>
							<td headers="h_M7">ens</td>
							<td headers="h_M9" contenteditable="true">1</td>
							<td headers="h_M18"></td>
							<td headers="h_M20"></td>
							<td headers="h_M14"></td>
							<td headers="h_M19"></td>
							<td headers="h_M23"></td>
							<td headers="h_M26"></td>
							<td headers="h_M22"></td>
							<td headers="h_M42"></td>
							<td headers="h_M40"></td>
							<td headers="h_M41"></td>
							<td headers="h_M27"></td>
							<td headers="h_M32"></td>
							<td headers="h_M44"></td>
						</tr>
					</tbody>
				</table>
			</div>
			<div class="tab-pane fade" id="profile" role="tabpanel" aria-labelledby="profile-tab">
				<div id="tab2">
					<table id="mytable2">
						<thead>
	
						</thead>
						<tbody>
							<tr>
	
							</tr>
						</tbody>
					</table>
				</div>
			</div>
			<div class="tab-pane fade" id="contact" role="tabpanel" aria-labelledby="contact-tab">...</div>
			</div>
		</div>

		<ul class="nav nav-tabs" id="myTab" role="tablist">
			<li class="nav-item" role="presentation">
				<button class="nav-link active" id="home-tab" data-bs-toggle="tab" data-bs-target="#home" type="button"
					role="tab" aria-controls="home" aria-selected="true">Home</button>
			</li>
			<li class="nav-item" role="presentation">
				<button class="nav-link" id="profile-tab" data-bs-toggle="tab" data-bs-target="#profile" type="button"
					role="tab" aria-controls="profile" aria-selected="false">Profile</button>
			</li>
			<li class="nav-item" role="presentation">
				<button class="nav-link" id="contact-tab" data-bs-toggle="tab" data-bs-target="#contact" type="button"
					role="tab" aria-controls="contact" aria-selected="false">Contact</button>
			</li>
		</ul>

	</main>

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
	<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.13.2/jquery-ui.min.js"></script>
	<script type="text/javascript" src="http://www.jeasyui.com/easyui/jquery.easyui.min.js"></script>
	<script src="https://cdn.jsdelivr.net/npm/bootstrap@3.4.1/dist/js/bootstrap.min.js"
		integrity="sha384-aJ21OjlMXNL5UyIl/XNwTMqvzeRMZH2w8c5cRVpzpU8Y5bApTppSuUkhZXN0VxHd"
		crossorigin="anonymous"></script>
	<script src="https://unpkg.com/bootstrap-table@1.21.0/dist/bootstrap-table.min.js"></script>
	<script src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.min.js"></script>
	<script src="https://cdn.datatables.net/colreorder/1.5.6/js/dataTables.colReorder.min.js"></script>
	<script type="text/javascript"
		src="https://cdn.datatables.net/responsive/2.3.0/js/dataTables.responsive.min.js"></script>
	<script src="https://cdn.datatables.net/1.12.1/js/dataTables.bootstrap.min.js"></script>
	<script src="voctor.js"></script>
</body>
</html>`

Here's the JS : 

`$(document).ready(function()
    var triggerTabList = [].slice.call(document.querySelectorAll('#myTab a'))
    triggerTabList.forEach(function (triggerEl) {
        var tabTrigger = new bootstrap.Tab(triggerEl)

        triggerEl.addEventListener('click', function (event) {
            event.preventDefault()
            tabTrigger.show()
        })
    })
})`

Thanks in advance.
Sincerely,
Victor

**Comment by Edditoria** [comment]
Please update your question with correct Markdown first. Thanks.
